### ISSUE TITLE
Create method to update ticker

### ISSUE THREAD

**Issue by jbaxleyiii** [NewSpring/celebrate.newspring.cc#15]
Currently to force and update to the ticker you have to reduce a week number then up it back to the previous amount.

We should make this a callable method

**Comment by delianides** [comment]
are you saying this should be a method in the ticker itself?

**Comment by jbaxleyiii** [comment]
@delianides no, a method via the meteor shell or possible an api endpoint we can hit

**Comment by delianides** [comment]
the ticker has a reset method so you can clear the current number then set the new number.

**Comment by jbaxleyiii** [comment]
@delianides that is what the app is using to update the number right now. I'm talking about a way to force a push from the app
